addons/im_livechat/static/tests/embed/livechat_service.test.js

```diff
@@ -129,6 +129,7 @@ test("Only necessary requests are made when creating a new chat", async () => {
                 body: "Hello!",
                 message_type: "comment",
                 subtype_xmlid: "mail.mt_comment",
+                email_add_signature: true,
```

Comment:
a bit everywhere: we try to keep them alphabetical as much as possible (it's not always done correctly)